packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -63,6 +64,21 @@ export default class PciStoragesContainersAddController {
 
   onTypesFocus() {
     this.displaySelectedType = false;
+    this.loadingPrice = true;
+    this.PciProjectStorageContainersService.getPriceEstimation(
+      this.user.ovhSubsidiary,
+    )
+      .then((price) => {
+        this.price = price;
+        if (this.price !== null) {
+          this.price.formatedPrice = this.getFormatedPrice(
+            price.price / 100000000,
```

Comment:
We might need to check we don't have a component/filter/service that doesn't already handle microcents prices

Reply:
I think we need to add a price format filter or component